### ISSUE TITLE
Non-auth users can create projects, etc?

### ISSUE THREAD

**Issue by Disastorm** [tolgee/tolgee-platform#1885]
Not sure if this is a bug or what, but when following the setup guide for docker, using the yaml and running with docker compose, upon startup, when you goto the page you are shown as "No auth user" and you can click add project and create a project. 

Is this the normal functionality of tolgee? According to the configuration docs, it should default to authentication enabled and yet there is no login button or anything and it lets anonymous users do stuff.

**Comment by JanCizmar** [comment]
You're right. The authentication is disabled by default only for docker. So it should be documented.

**Comment by Disastorm** [comment]
thanks for confirming. sorry a bit of a different question here but do you know how i can actually log in? The config has a username and password section but the login page asks for an email and password, and putting the username into the email field along with the password doesn't seem to work.

**Comment by cyyynthia** [comment]
This is a current bug I'm working on fixing along with other changes very soon; because Tolgee created a ghost user to allow the "no-authentication" workflow to function, the initial user is not getting created because Tolgee thinks there's already a user registered (the ghost user).

If you haven't created any projects and you're just getting started, clear the volume associated with the PostgreSQL container to clear the database, and start everything back up fresh. The initial user will be created.

And yes, it's a bit confusing but you should enter the username in the email field. The name conflict comes from legacy heritage from very oooooold Tolgee versions where it was an actual username rather than an email - the field got reused for emails later on but the naming of things remained unchanged to avoid breaking changes.

I'll also drop a note about that in the documentation, along with specifying that authentication is disabled by default in the Docker image. If you have any additional remarks or suggestions regarding the authentication experience with Tolgee, let us know! :D

**Comment by Disastorm** [comment]
thanks confirmed clearing the data fixed it.

**Comment by AlienHub** [comment]
> thanks confirmed clearing the data fixed it.

I want to ask how to reset the database, I can not connect to him

**Comment by JanCizmar** [comment]
@AlienHub 

> I want to ask how to reset the database, I can not connect to him

It depends on how are you deploying it. Do you use docer, docker compose, java? Can you share your config?
